File: 2st/Yeonwoo/코드 처리하기.py
def solution(code):
    answer = ''
    mode = 0
    for i, co in enumerate(code):
        if mode == 0:
            if co!='1' and i%2==0:
                answer += co
            elif co=='1':
                mode = 1
        else:
            if co!='1' and i%2==1:
                answer += co
            elif co=='1':
                mode = 0
    if answer == '':
        answer = 'EMPTY'
    return answer

File: 4st/Gau/programmer/코드 처리하기.java
class Solution {
    public String solution(String code) {
        String answer = "";
        int mode = 0;

        for(int i=0;i<code.length();i++){
            if(mode==0){
               if(code.charAt(i)!='1'){
                    if(i%2==0){
                        answer+=code.charAt(i);
                    }
                }else if(code.charAt(i)=='1'){
                    mode=1;
                } 
            }else{
                if(code.charAt(i)!='1'){
                    if(i%2==1){
                        answer+=code.charAt(i);
                    }
                }else if(code.charAt(i)=='1'){
                    mode=0;
                }
            }

        }

        if(answer.equals("")){
            answer = "EMPTY";
        }
        return answer;
    }
}